src/components/ButtonPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Button from './Button';
+
+class ButtonPannel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
```

Comment:
This can also be a function component, for this project only `App` component needs to be a stateful component.